### ISSUE TITLE
Add support for LocalDate and java.util.Date 

### ISSUE THREAD

**Issue by seslav** [vert-x3/vertx-codegen#137]
I'm using codegen to generate some proxy services to persist data in database. But as mention in the documentation Date and LocalDate should have DataObject annotation in order service to work. The problem is that LocalDate cannot be extended, java.util.Date give this opportunity but still not very clear solution.
Can you add support for these two class types or advise how to work around the issue?

**Comment by vietj** [comment]
I think that you could handle them yourself using `@GenIgnore` annotation and marshall that in json object converting the date to instant

```
@GenIgnore
public void setDate(Date date) {
   jsonObject.put("date", date.toInstant())
}
```

**Comment by seslav** [comment]
I made a sample project with the idea I'm trying to achieve - https://github.com/seslav/vertx-localdate/tree/master/src/main/java/localdatetest
`App.java` is the main verticle that loads HttpVerticle and creates DataService proxy service that listens on event bus. 
There is an Person class that has name and date of birth (`dob`). What i'm trying is via rest call to **HTTP POST /person** to persist the object. Unfortunately in `DataServiceImpl` persist method the person object comes with empty `dob` field. Dob field is empty because the converter is ignoring the `LocalDate` object. 

Now I'm thinking that I didn't explain well my question (sorry about that). I'm trying to pass `LocalDate` object through eventbus to a service that will persist the data in database and not to convert `Date` object to `LocalDate`.

Example
HTTP POST `localhost:4001/person`
Content-type: application/json
Body 
`{
	"name":"Vertx",
	"dob":"12-12-2017"
}`

I found a workaround by using `Json.decodeValue(json.toString(), Person.class);` instead of `PersonConverter`.
